prettier.config.mjs

```diff
@@ -10,6 +10,5 @@ export default {
   tabWidth: 2,
   printWidth: 80,
   arrowParens: 'always',
-  editorconfig: true,
```

Comment:
removed as this rule is actually unknown by prettier, it got revealed after running prettier in cli